package-build/tools/checkEnvironment.ps1

```diff
@@ -0,0 +1,23 @@
+$ErrorActionPreference = 'Stop';
+
+Write-Host 'Checking environment...'
+
+# Check system PATH length
+# https://support.microsoft.com/en-ca/help/830473/command-prompt-cmd-exe-command-line-string-limitation
+$PathLen = $Env:PATH.Length
+$MaxPathLen = 8191
+if ($PathLen -gt $MaxPathLen) {
+    Write-Warning "The length of %PATH% is over $MaxPathLen."
+    Write-Warning "Reduce the %PATH% length and retry again."
+    throw "The length of %PATH% is too long."
+}
+
+# Check git existence
+$gitInstalled = Get-Command -ErrorAction SilentlyContinue git
+if (-not $gitInstalled) {
+    Write-Warning "Git is required to proceed."
```

Comment:
"In order to install ROS on this system, Git source control command line interface is required. At the moment, it is not detected. This is either because it hasn't been installed in a previous step, or this environment has not been updated to find it. Please reinstall using `choco install git -y` or if you have done that, restart this command window"